reconcile/utils/helm.py

```diff
@@ -46,4 +51,26 @@ def template(values: Mapping[str, Any]) -> Mapping[str, Any]:
             msg += f" {e.stderr.decode()}"
         raise HelmTemplateError(msg)
 
-    return yaml.safe_load(result.stdout.decode())
+    return result.stdout.decode()
+
+
+def template(
+    values: Mapping[str, Any],
+    path: str = "./helm/qontract-reconcile",
+    name: str = "qontract-reconcile",
+) -> Mapping[str, Any]:
+    return yaml.safe_load(do_template(values=values, path=path, name=name))
+
+
+def template_all(
+    url: str,
+    path: str,
+    name: str,
+    values: Mapping[str, Any],
+    ssl_verify: bool = True,
+) -> Iterable[Mapping[str, Any]]:
+    with tempfile.TemporaryDirectory() as wd:
+        git.clone(url, wd, verify=ssl_verify)
```

Comment:
since we just need files, shallow clone is enough (depth=1)

Reply:
https://github.com/app-sre/qontract-reconcile/pull/4472/commits/b0685754fe50c1edcb9c5e83bc47d64c7a51cc48